dspace/src/main/config/build.xml

```diff
@@ -748,7 +748,7 @@ Common usage:
 
         <mkdir dir="${handle.dir}" />
 
-        <mkdir dir="${log.dir}" />
+        <mkdir dir="${dspace.dir}/log" />
```

Comment:
This works fine for our default configuration, but wouldn't it be better to use the log.dir value out of log4j.properties? I know that this works a little bit against the idea of getting the log4j configuration as independent as possible, but if someone changes the log.dir in any of the log4j*.properties files and forgets to create the appropriate directory, he/she ends up with an invalid log4j configuration.

Alternatively we could leave this as it is and add a comment to the "log.dir" property in all log4j*.properties files that anyone who changes this, must ensure that the directory exists and is writeable.